src/App.tsx

```diff
@@ -15,71 +20,86 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  handleResetClick = () => {
```

Comment:
looks simpler and does not lose any of the clarity
```suggestion
  handleReset = () => {
```